docs/source/developers/guide/communication.rst

```diff
@@ -0,0 +1,116 @@
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+
+.. SCOPE OF THIS SECTION
+.. Present Arrow developers to the general public to make
+.. the barrier of contributing lower as new contributors will
+.. have a feeling of joint work and possible help. Add ways
+.. of communication and add description of what is expected.
+.. Also add a link to other resources.
+
+
+.. _communication:
+
+*************
+Communication 
+*************
+
+**About the contributors**
+
+The group of contributors is full of experts, software engineers and core
+developers but also full of users, learners and enthusiasts that love doing
+what we do and we want to spread this enthusiasm to you also.
+
+We all have silly questions and we all need help lots of times.
+We encourage you to be open to communication and we will help as much as
+we can.
+
+Due to the scale of the project which includes many languages, everybody
+contributing will be faced with new things needed to be learned. Even the most
+seasoned C++ developer will need to ask basic questions about R for example.
+
+When communicating, it's important you mark your communication with appropriate
+tags ([C++], [R], [Ruby] etc.) so it gets noticed by the right people.
+
+Where to get help 👋
+====================
+
+JIRA
+~~~~
+
+The most used way is to communicate through JIRA.
+
+When to use JIRA?
+
+- If you want to **report a bug**.
+- If you want to **propose a new feature**.
+- If you want to **propose a bigger change in the documentation**.
+
+In these cases you can create a JIRA issue and connect
+with other contributors in the comments.
+
+.. note::
+   How to make a JIRA issue. See `Tips for using JIRA <https://arrow.apache.org/docs/developers/contributing.html#tips-for-using-jira/>`_.
+
+- If you want to **solve an issue that is already in JIRA**, you should
+  connect with other contributors in the comments.
+
+JIRA is used to discuss the problem before the pull request is made (how to start,
+where to search for the solution, etc..).
+
+GitHub
+~~~~~~
+
+Further options of communicating are provided through GitHub.
+
+When to use GitHub?
+
+- **For any question you may have or problems you are facing, you can make an issue on GutHub.**
+- If you have **a problem with the build**, you can make an issue on GitHub
+  and discuss the possible solution (or write to the user mailing list).
+
+  Making an issue about things you are not sure about may feel intimidating,
+  but it is also useful for others and the project.
+
+  .. warning::
+     Make sure to add which operating system and Arrow version you are using in the
+     ticket description plus the debug information/error.
+
+- If you have **a new feature already written**, you should make a pull request and
```

Comment:
Hmm, but in this case I would still expect the contributor to create a JIRA first.